.travis.yml

```diff
@@ -12,14 +12,14 @@ before_install:
   - sudo sysctl -w vm.max_map_count=262144
   - sudo service docker restart
   - docker pull centos:7
+  - docker pull crate/crate:4.0.10
```

Comment:
Why is this now necessary?

Reply:
I will check, whether it is now. The image for version 4.0.10 did not exist and the tests failed until I pulled it. Now that it is pulled I can remove it I suppose.

Reply:
Computer says no: ```docker.errors.ImageNotFound: 404 Client Error: Not Found ("No such image: crate/crate:4.0.10")
```